torch_glow/src/FusingOptimizer.cpp

```diff
@@ -89,4 +92,16 @@ graph(%input, %weight):
   mm_to_linear.runOnGraph(graph);
 }
 
+void glowCustomFuse(std::shared_ptr<torch::jit::Graph> &g,
```

Comment:
We should eventually split fusions here into different files, can this go into PyTorchCommon.cpp?